base/src/main/java/io/spine/protobuf/ProtoConverter.java

```diff
@@ -21,40 +21,53 @@
 package io.spine.protobuf;
 
 import com.google.common.base.Converter;
-import com.google.common.base.Function;
 import com.google.protobuf.ByteString;
 import com.google.protobuf.Message;
 import com.google.protobuf.ProtocolMessageEnum;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
 /**
- * The {@link Function} performing the described type conversion.
+ * Performs two-way conversion of a {@linkplain Message Protobuf Message} to its
+ * Java counterpart and back.
+ *
+ * <p>The inheritors implement the actual conversion to a specific Message and Object.
  *
  * @param <M>
- *         type of message to convert
+ *         Protobuf Message to convert
```

Comment:
It's not the message, but its type. How about “the type of a message to convert"?